src/appleseed/renderer/modeling/bsdf/bsdfsample.h

```diff
@@ -99,6 +101,7 @@ inline BSDFSample::BSDFSample(
   , m_shading_basis(shading_point->get_shading_basis())
   , m_outgoing(outgoing)
   , m_mode(ScatteringMode::None)
+  , m_is_refraction_sample(false)
```

Comment:
It is really important to initialize members in the same order as they are declared. Here it has no consequences but in other places it's critical to do so, so best keep the habit.